features/bonding/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
i think this file is not needed

Reply:
completely forgot to exclude it, removed, thanks!